GitHub Pages workflow is failing

The `.github/workflow/sphinx_docx_to_gh_pages.yaml` is failing with the following...

```
Run pip3 install git+https://github.com/Dzordzu/sphinx-markdown-tables.git@markdown-patch
Collecting git+https://github.com/Dzordzu/sphinx-markdown-tables.git@markdown-patch
  Cloning https://github.com/Dzordzu/sphinx-markdown-tables.git (to revision markdown-patch) to /tmp/pip-req-build-qk6wbyl1
  Running command git clone --filter=blob:none --quiet https://github.com/Dzordzu/sphinx-markdown-tables.git /tmp/pip-req-build-qk6wbyl1
  fatal: could not read Username for 'https://github.com': No such device or address
  error: subprocess-exited-with-error
  
  × git clone --filter=blob:none --quiet https://github.com/Dzordzu/sphinx-markdown-tables.git /tmp/pip-req-build-qk6wbyl1 did not run successfully.
  │ exit code: 12[8](https://github.com/claritychallenge/clarity/actions/runs/3461853906/jobs/5780008195#step:4:9)
  ╰─> See above for output.
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× git clone --filter=blob:none --quiet https://github.com/Dzordzu/sphinx-markdown-tables.git /tmp/pip-req-build-qk6wbyl1 did not run successfully.
│ exit code: [12](https://github.com/claritychallenge/clarity/actions/runs/3461853906/jobs/5780008195#step:4:13)8
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.

Notice:  A new release of pip available: [22](https://github.com/claritychallenge/clarity/actions/runs/3461853906/jobs/5780008195#step:4:23).3 -> 22.3.1
Notice:  To update, run: pip install --upgrade pip
Error: Process completed with exit code 1.
```

This is because there was an error a while back with `sphinx-markdown-tables` which `Dzordzu` fixed with a patch. That has now been [merged](https://github.com/Slicer/Slicer/pull/6474) and the current `sphinx-markdown-tables-0.0.17` should work.

The line `pip3 install git+https://github.com/Dzordzu/sphinx-markdown-tables.git@markdown-patch` can now be removed as the required package is listed under the `docs` requirements.

--- a/.github/workflows/sphinx_docx_to_gh_pages.yaml
+++ b/.github/workflows/sphinx_docx_to_gh_pages.yaml
@@ -20,7 +20,6 @@ jobs:
           python-version: 3.9
       - name: Installing the Documentation requirements
         run: |
-          pip3 install git+https://github.com/Dzordzu/sphinx-markdown-tables.git@markdown-patch
           pip3 install .[docs]
       - name: Running Sphinx to gh-pages Action
         uses: ns-rse/action-sphinx-docs-to-gh-pages@main

